src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.cs

```diff
@@ -435,6 +436,29 @@ private void OnTextWrappingChanged(DependencyPropertyChangedEventArgs args)
 
 		#endregion
 
+		public CharacterCasing CharacterCasing
+		{
+			get => (CharacterCasing)this.GetValue(CharacterCasingProperty);
+			set => this.SetValue(CharacterCasingProperty, value);
+		}
+
+		public static DependencyProperty CharacterCasingProperty { get; } =
```

Comment:
We should still keep the NotImplemented attribute for this and the property above for platforms that do not support it.